src/components/v5/frame/ColonyHome/partials/TotalInOutBalance/utils.ts

```diff
@@ -0,0 +1,48 @@
+import { format, setMonth } from 'date-fns';
+import numbro from 'numbro';
+
+import { convertToDecimal } from '~utils/convertToDecimal.ts';
+
+import { type BarChartDataItem } from './types.ts';
+
+export const sortByLabel = (a: BarChartDataItem, b: BarChartDataItem) =>
+  parseInt(a.label ?? '', 10) - parseInt(b.label ?? '', 10);
```

Comment:
This seems to be quite dangerous since `parseInt('', 10)` results in `NaN`

Maybe you can use `parseInt(a.label ?? 0, 10)` instead?